probe-rs/src/debug/source_instructions.rs

```diff
@@ -57,10 +57,13 @@ impl VerifiedBreakpoint {
     ///     we need to find the closest "relevant" source location that has valid debug information.
     ///   - The requested location may not be a valid source location, e.g. when the
     ///     debug information has been optimized away. In this case we will return an appropriate error.
+    ///
     /// #### The logic used to find the "most relevant" source location is as follows:
-    /// 1. Filter  [`UnitInfo`] , by using [`LineProgramHeader`] to match units that include the requested path.
-    /// 2. For each matching compilation unit, get the [`LineProgram`] and [`Vec<LineSequence>`].
-    /// 3. Filter the [`Vec<LineSequence>`] entries to only include sequences that match the requested path.
+    /// 1. Filter  [`UnitInfo`], by using [`LineProgramHeader`][gimli::LineProgramHeader] to match units that include
```

Comment:
I think I'd put these as ``[`gimli::X`]`` so it's obvious these types are external to probe-rs.